### PR TITLE
Document `pubsub_id` param of `request_missing_blocks`

### PR DESCRIPTION
**Pull request by hrxi**

### CODE REVIEW

File: consensus/src/sync/live/block_queue/mod.rs [hunk 1]
```diff
@@ -272,6 +272,10 @@ impl<N: Network> BlockQueue<N> {
     /// Requests missing blocks.
     /// If a block locator is given, it is inserted at the beginning of the full block locator list.
     /// The list contains all blocks from the head until the last macro block.
+    ///
+    /// `pubsub_id` specifies the gossipsub ID that of the gossip that led to
```
Comment (jsdanielh) [inline consensus/src/sync/live/block_queue/mod.rs]:
```suggestion
    /// `pubsub_id` specifies the gossipsub ID that of the peer that led to
```

Comment (hrxi) [reply consensus/src/sync/live/block_queue/mod.rs]: Maybe the "gossipsub peer ID of the gossip message that led to…"? It's the message leading to the request, not the peer.

Comment (jsdanielh) [reply consensus/src/sync/live/block_queue/mod.rs]: Ah, that makes sense

Comment (hrxi) [reply consensus/src/sync/live/block_queue/mod.rs]: Fixed.

**Review by jsdanielh — APPROVED** [review]

**Comment by sisou** [comment]
Alternatively, you could have renamed the argument to `pubsub_peer_id`, putting the "peer" meaning of it in the name and removing the need to document it so detailed.

**Comment by hrxi** [comment]
That would still not be clear enough to me. "Why do I need a pubsub peer ID for requesting missing blocks? What happens if I set it to `None`?"

**Pull request merged**